replication-api-impl/src/main/java/com/connexta/replication/api/impl/spring/ServiceConfig.java

```diff
@@ -85,6 +91,26 @@ public NodeAdapters nodeAdapters(List<NodeAdapterFactory> nodeAdapterFactories)
     return nodeAdapters;
   }
 
+  @Profile("Ion")
+  @Bean
+  public QueueBroker queueBroker(MeterRegistry meterRegistry) {
+    return new MemoryQueueBroker(20, meterRegistry);
```

Comment:
✏️ I think we can use a much higher capacity than this. I would bump this up into the 1000s